src/main/scala/io/citrine/lolo/validation/Merit.scala

```diff
@@ -215,3 +275,95 @@ object Merit {
     chart
   }
 }
+
+/**
+  * A single predicted-vs-actual point in two real-valued dimensions.
+  * The covariance matrix of the prediction has diagonals sigmaX**2 and sigmaY**2, and off-diagonals rho*sigmaX*sigmaY
+  *
+  * @param dx the error along the first dimension
+  * @param dy the error along the second dimension
+  * @param sigmaX the uncertainty along the first dimension
+  * @param sigmaY the uncertainty along the second dimension
+  * @param rho the correlation coefficient
+  */
+case class PredictedVsActualTwoDimensions(
```

Comment:
It seems like this could generalize with the Mahalanobis distance to N-dimensions quite easily. Would it make sense to test >2 dimensions for some of your upcoming examples?

Reply:
It might, since a real-world example would involve predicting the probability of simultaneously hitting N targets, where N could be higher than 2.

The generalization is straightforward, which is good. It might be a bit hairy to implement, but I think it's worth trying.